### PR TITLE
Add `create_button_label` option to live resource

### LINKED ISSUE #382: Add configuration for create button label
The label for the create button should be configurable. This also includes the create button in the empty state.

- [ ] Add a `create_button_label` function / option to live resources
- [ ] Add guide for new feature

### PR DESCRIPTION
**Pull request by Flo0807**
closes #382

### CODE REVIEW

File: lib/backpex/live_resource.ex [hunk 1]
```diff
@@ -121,6 +121,11 @@ defmodule Backpex.LiveResource do
   @callback return_to(socket :: Phoenix.LiveView.Socket.t(), assigns :: map(), action :: atom(), item :: map()) ::
               binary()
 
+  @doc """
+  Customizes the label of the button for creating a new item. Defaults to "New %{resource}".
+  """
+  @callback create_button_label() :: binary()
```
Comment (Flo0807) [inline lib/backpex/live_resource.ex:127]: Should we pass certain parameters to the callback function, e.g. the `assigns`?

File: mix.exs [hunk 2]
```diff
@@ -106,6 +106,7 @@ defmodule Backpex.MixProject do
       "guides/live_resource/panels.md",
       "guides/live_resource/fluid-layout.md",
       "guides/live_resource/listen-to-pubsub-events.md",
+      "guides/live_resource/custom_button_labels.md",
```
Comment (pehbehbeh) [inline mix.exs:109]: We have a lot of short really specific custom guides. Maybe combine with translations?

File: lib/backpex/live_resource.ex (hunk 1 above)
Comment (pehbehbeh) [reply lib/backpex/live_resource.ex:127]: Let's add it when we need it? (YAGNI)

**Review by pehbehbeh — APPROVED** [review]

File: mix.exs (hunk 2 above)
Comment (Flo0807) [reply mix.exs:109]: Moved it to translations and renamed translations to "Custom Labels and Translations"

**Pull request merged**